executor/src/test/java/com/redhat/service/bridge/executor/ExecutorTest.java

```diff
@@ -73,14 +82,40 @@ public void testOnEventWithFiltersTransformationAndAction() throws JsonProcessin
         BaseAction action = new BaseAction();
         action.setType(KafkaTopicAction.TYPE);
 
-        ProcessorDTO processorDTO = createProcessor(filters, transformationTemplate, action);
+        ProcessorDTO processorDTO = createProcessor(new ProcessorDefinition(filters, transformationTemplate, action));
+
+        Executor executor = new Executor(processorDTO, filterEvaluatorFactory, transformationEvaluatorFactory, actionProviderFactoryMock, meterRegistry);
+
+        CloudEvent cloudEvent = createCloudEvent();
+
+        executor.onEvent(cloudEvent);
+
+        verify(actionProviderFactoryMock).getInvokableActionProvider(KafkaTopicAction.TYPE);
+        verify(actionInvokerMock).onEvent(any());
+    }
+
+    @Test
+    public void testOnEventWithFiltersTransformationActionAndVirtualAction() throws JsonProcessingException {
```

Comment:
We're not using the term "virtualAction" any more, so can you rename this method to something more appropriate.

Reply:
Fixed :+1: